src/main/resources/common_at.cfg

```diff
@@ -179,8 +179,20 @@ public-f net.minecraft.server.management.UserList *
 public net.minecraft.server.management.UserList func_152688_e()Ljava/util/Map;
 public-f net.minecraft.server.management.UserListEntry field_152642_a
 
+public net.minecraft.tileentity.MobSpawnerBaseLogic field_98288_a # mobID
 public net.minecraft.tileentity.MobSpawnerBaseLogic field_98286_b # spawnDelay
+public net.minecraft.tileentity.MobSpawnerBaseLogic field_98285_e # minecartToSpawn
+public net.minecraft.tileentity.MobSpawnerBaseLogic field_98282_f # randomEntity
+public net.minecraft.tileentity.MobSpawnerBaseLogic field_98283_g # minSpawnDelay
+public net.minecraft.tileentity.MobSpawnerBaseLogic field_98293_h # maxSpawnDelay
+public net.minecraft.tileentity.MobSpawnerBaseLogic field_98294_i # spawnCount
 public net.minecraft.tileentity.MobSpawnerBaseLogic field_98292_k # maxNearbyEntities
+public net.minecraft.tileentity.MobSpawnerBaseLogic field_98289_l # activatingRangeFromPlayer
+public net.minecraft.tileentity.MobSpawnerBaseLogic field_98290_m # spawnRange
+public net.minecraft.tileentity.MobSpawnerBaseLogic func_98276_e()Ljava/lang/String; # getEntityNameToSpawn
+public net.minecraft.tileentity.MobSpawnerBaseLogic func_180613_a(Lnet/minecraft/entity/Entity;Z)Lnet/minecraft/entity/Entity; # spawnNewEntity
+public net.minecraft.tileentity.MobSpawnerBaseLogic$WeightedRandomMinecart field_98222_b # nbtData
+public net.minecraft.tileentity.MobSpawnerBaseLogic$WeightedRandomMinecart field_98223_c # entityType
```

Comment:
If possible use existing getters/setters (unless they have unintended side effects), but you dont have to create them if they dont exist.